src/backend/wiki_game_info/protondb/utils.ts

```diff
@@ -0,0 +1,35 @@
+import { ProtonDBInfo, ProtonDBCompatibilityInfo } from 'common/types'
+import axios, { AxiosError } from 'axios'
+import { logDebug, logError, LogPrefix } from 'backend/logger/logger'
+
+export async function getInfoFromProtonDB(
+  steamID: string
+): Promise<ProtonDBCompatibilityInfo | null> {
+  if (steamID === '') {
+    logDebug('No SteamID, not getting ProtonDB info')
+    return null
+  }
+  const url = `https://www.protondb.com/api/v1/reports/summaries/${steamID}.json`
+
+  const response = await axios
+    .get<ProtonDBInfo>(url, { headers: {} })
```

Comment:
To use `ProtonDBInfo` as name for heroic probably make this type used in axio some kind of temp type if possible.
Also will this break if protondb decide to add a new field from one day to another? Better fetch without type and check which fields are there?